cmake/MOAB_PullAndMake.cmake

```
@@ -0,0 +1,45 @@
+# this Macro sets up the download and build of MOAB using ExternalProject
+# few tweak are done in src/dagmc/CMakeLists.txt and src/PyNE/CMakelists.txt 
+# to make sure that MOAB is built before DAGMC.
+MACRO (moab_pull_make moab_version)
+    message(STATUS "MOAB will be downloaded and built")
+    include(ExternalProject)
+    message("HDF5_ROOT: ${HDF5_ROOT}")
+    SET(MOAB_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/moab")
```

should this be lower case like the others
```suggestion
    set(MOAB_INSTALL_PREFIX "${CMAKE_INSTALL_PREFIX}/moab")
```

---

You are probably right, I added your suggestions.

I usually prefer to keep `CMake` instructions in all caps, this explains the confusion. and the strange `MEsSAGE`